roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -47,10 +47,12 @@
 - name: Update server field in kubelet kubeconfig
   replace:
     path: "{{ kube_config_dir }}/kubelet.conf"
-    regexp: '(\s+)https://{{ first_kube_master }}:{{ kube_apiserver_port }}(\s+.*)?$'
+    regexp: '(\s+)https://{{ hostvars[item].ansible_default_ipv4.address }}:{{ kube_apiserver_port }}(\s+.*)?$'
```

Comment:
it's not always the default_ipv4 that is used

Reply:
```
{{ hostvars[item]['access_ip'] | default(hostvars[item]['ip'] | default(hostvars[item]['ansible_default_ipv4']['address'])) }}
```

Reply:
Thank's for your review. I made the requested changes.